Stock_Alert/main.py

```diff
@@ -1,74 +1,66 @@
-import requests
+from response import Response
+from decimal import Decimal
 import os
 
 STOCK_SYMBOL = "OTLY"
 COMPANY_NAME = "Oatly"
 
 STOCK_ENDPOINT = "https://www.alphavantage.co/query"
-NEWS_ENDPOINT = "https://newsapi.org/v2/everything"
-
 stock_api_key = os.environ.get("STOCK_API_KEY")
```

Comment:
(minor) btw Readme would benefit from clear instructions on how to get  those API keys

Reply:
You'r right, sorry for any inconvenience. I refactored it, hope it is now clearer.